msteams-platform/m365-apps/combine-office-add-in-and-teams-app.md

```diff
@@ -0,0 +1,621 @@
+---
+title: Add an Outlook Add-in to a Teams app
+description: Learn how to add an Outlook Add-in to a Teams app. This article will walk you through the steps to add an Outlook Add-in to a Teams app, test it, then deploy it to Azure.
+ms.date: 02/15/2024
+ms.author: mosdevdocs
+author: rickki
+ms.topic: tutorial
+ms.localizationpriority: medium
+ms.subservice: m365apps
+---
+
+# Add an Outlook Add-in to a Teams app
+
+## Introduction
+
+[Outlook Add-ins](/office/dev/add-ins/outlook/outlook-add-ins-overview) are web apps that extend the functionality of Outlook. With an Outlook Add-in, you can:
+
+- Read and write email messages and engage in meeting invitations, responses, cancellations, and appointments.
+- Read properties of the user's mailbox.
+- Respond to events automatically, such as sending an email.
+- Integrate with external services including CRM and project management tools.
+- Add custom ribbon buttons or menu items to perform specific tasks.
+
+For more info, [Outlook Add-ins Overview](/office/dev/add-ins/outlook/outlook-add-ins-overview) provides information about what you can do with Outlook Add-ins, how they are structured, how they differ from older ways of extending Outlook, what platforms the add-in can run on (Windows, Mac, mobile, and the web), and how to get started creating one.
+
+Combining an Outlook Add-in with a Teams app enables scenarios that can't be satisfied with either of them separately. For example, consider an app that enables a salesperson to insert a customized discount in an email to a customer and records the offer in a backend database. The sales manager can also view data about all the discounts that have been offered in a Teams tab. For more information, see [Discount Offers sample](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/tab-add-in-combined/nodejs).
+
+This article will walk you through the steps to add an Outlook Add-in to a Teams app, test it, then deploy it to Azure.
+
+## Prerequisites 
+
+To configure an Office Add-in, ensure that you meet the following requirements: 
+
+- A Microsoft 365 account that includes Teams to test the appl. Alternatively, you can have separate subscriptions to both Microsoft 365 and Teams. For example, a test account with *.onmicrosoft.com through the [Microsoft 365 Developer Program](/office/developer-program/microsoft-365-developer-program). 
```

Comment:
Oops, was just about to approve--spotted this typo:

A Microsoft 365 account that includes Teams to test the **appl**.

Reply:
> Oops, was just about to approve--spotted this typo:
> 
> A Microsoft 365 account that includes Teams to test the **appl**.

Fixed